### ISSUE TITLE
0.24.0-beta-5 creates invalid unicode escape sequence

### ISSUE THREAD

**Issue by Syer10** [icerockdev/moko-resources#723]
After updating to beta-5 from beta-4, a new error popped up with our translated files.
```
> Task :app:mergeDebugResources
J:\repo\app\build\intermediates\incremental\debug\mergeDebugResources\merged.dir\values-de\values-de.xml:238:61: Failed to flatten XML for resource 'android_something' with error: Invalid unicode escape sequence in string
"{str}"
J:\repo\strings\build\intermediates\packaged_res\debug\values-de\values-de.xml:207:4: string/android_something does not contain a valid string resource.
```

The `de/strings.xml` file contains this string
```xml
<string name="android_something">Something \'Anrufe tätigen\' something</string>

```
Which in the merged file turns into this `merged.dir\values-de\values-de.xml`
```xml
<string name="android_something">Something \\'Anrufe tätigen\\' something</string>
```

**Comment by Syer10** [comment]
These are translated files that come directly from our translation service that are meant to be directly compatible with android resources, and Android requires that `'` should be escaped so the translation service escapes them all

**Comment by ExNDY** [comment]
Please, remove mirror for quote, on current time generator create correct resources for all targets and mirroring need only in exclusive situation

**Comment by Syer10** [comment]
The files themselves are fetched directly from the service in CI, we would have to implement a pre-processing step after the files are fetched but before they are added to the repo. I feel like this should be handled here since Moko is mean to support Android string files

**Comment by ExNDY** [comment]
Moko is support generation for Android target, but never says about android strings format in localization strings. 

@Alex009 what's you think about this?

**Comment by Syer10** [comment]
A whole bunch of open source projects use Weblate as a automated translation service to translate Moko files, after this update Weblate would be unusable because it does the same thing.

**Comment by Alex009** [comment]
@ExNDY let's add processing of this case too

**Comment by Alex009** [comment]
will be released in 0.24.0

**Comment by Syer10** [comment]
Thank you!
